package.json

```
@@ -7,6 +7,7 @@
   },
   "scripts": {
     "start": "backstage-cli repo start",
+    "start:app-migrated": "yarn start app-migrated backend",
```

```suggestion
    "start:app-migrated": "backstage-cli repo start app-migrated backend",
```

matter of taste but, is this clearer?